test/e2e_node/runner/remote/run_remote.go

```diff
@@ -324,7 +318,7 @@ func main() {
 	running := 0
 	for shortName := range gceImages.images {
 		imageConfig := gceImages.images[shortName]
-		fmt.Printf("Initializing e2e tests using image %s.\n", shortName)
+		fmt.Printf("Initializing e2e tests using image %s/%s/%s.\n", shortName, imageConfig.project, imageConfig.image)
```

Comment:
so at the point this is printed, we've already made the choice of which image to run, and are getting ready to run it, awesome!

Reply:
yes, images and software version are important for troubleshooting. We should print as detailed as possible.